### ISSUE TITLE
Override duplex settings

### ISSUE THREAD

**Issue by felinira** [harwey/cups4j#60]
When settings duplex to false the printer default options are used. This makes it impossible to unset duplex for printers that have duplex as a default option in lpoptions.

It would be nice if it were possible to unset duplex mode by setting duplex to false, in the same way that color and page format work. This would add `sides:keyword:one-sided` to the list of options passed to cups.

**Comment by gmuth** [comment]
WIth [ipp-client](https://github.com/gmuth/ipp-client-kotlin) you'd use attribute [`Sides.OneSided`](https://github.com/gmuth/ipp-client-kotlin/blob/master/src/main/kotlin/de/gmuth/ipp/attributes/Sides.kt).

**Comment by DarrenCHoffman** [comment]
> When setting duplex to false the printer default options are used. This makes it impossible to unset duplex for printers that have duplex as a default option in lpoptions.
> 
> It would be nice if it were possible to unset duplex mode by setting duplex to false, in the same way that color and page format work. This would add `sides:keyword:one-sided` to the list of options passed to cups.


setting the PrintJob's duplex boolean to false doesn't do anything, but adding "sides:keyword:one-sided" as one of the "job-attributes" values in the attributes map does work.

**Comment by gmuth** [comment]
Another option is to change the default duplex setting in the printer settings to 'false'. This works for example on Xerox B210.

**Comment by DarrenCHoffman** [comment]
yes, that is true. But some of my customers for some reason want the default on their printer to be duplex but still have the option to print single sided to the same printer through our app. I think it's possible to also add the same printer more than once with different default settings though. customer has the last word though.
